R/vector-chr.R

```diff
@@ -82,7 +99,7 @@ string <- function(x, encoding = NULL) {
 #' as_bytes(latin1)
 #' }
 as_utf8_character <- function(x) {
-  enc2utf8(as_character(x))
+  .Call(rlang_unescape_character, as_character(x))
```

Comment:
This looks like a very far-reaching design choice. Maybe we should provide a separate operation for unescaping?